tests/e2e/state.go

```diff
@@ -111,6 +111,7 @@ type Param struct {
 func (tr TestRun) getState(modelState State) State {
 	systemState := State{}
 	for k, modelState := range modelState {
+		log.Println("Getting model state for chain: ", k)
```

Comment:
Is it worth standardizing between `log.Println` and `fmt.Println` in e2e tests? I see both being used

Reply:
Thanks, that's a good point. I think we should use log.Println whenever possible, my understanding is that it adds extra information like the timestamp. Not sure if it's worth refactoring existing stuff, but for new stuff it seems desirable.